src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -584,10 +584,20 @@ public enum LegendsAwakened implements LogicCardInfo {
             text "If you have Poliwag, Poliwhirl, and Poliwrath in play, each of these Pokémon's attacks does 60 more damage to the Defending Pokémon (before applying Weakness and Resistance)."
             delayedA {
               after PROCESS_ATTACK_EFFECTS, {
-                bg.dm().each {
-                  if (it.from.name in ["Poliwag", "Poliwhirl", "Poliwrath"] && it.to.active && it.to.owner != self.owner && it.dmg.value && it.notNoEffect) {
-                    bc "Enthusiasm +60"
-                    it.dmg += hp(60)
+                if (ef.attacker.owner == self.owner) bg.dm().each {
```

Comment:
This looks like a broken if statement

Reply:
?
Seems to compile fine, it was originally a one-line-if (just with a horrible multi-line condition in the middle)

There is a broken check inside though, probably due to not using `return` inside enthusiasm_cond, will send it in a minute.

Reply:
I mean, it might compile, but it still _looks_ wrong. Maybe you could put the `bg.dm().each {` on the next line to make it easier to read. As it is, it's definitely easy to read past the fact it is doing anything with the damage manager if you are used to `if` at beginning of line, `{` at end of line = "everything in between is conditions"

Reply:
That's fair, will move the each into a separate line. Usually move it, even if not adding {} brackets for the single-line if.